src/Files.App/ViewModels/MainPageViewModel.cs

```diff
@@ -224,12 +224,18 @@ public async Task<(string tabLocationHeader, Microsoft.UI.Xaml.Controls.IconSour
 				// If localized string is empty use the library name.
 				tabLocationHeader = string.IsNullOrEmpty(libName) ? library.Text : libName;
 			}
+			else if (App.WSLDistroManager.TryGetDistro(currentPath, out WslDistroItem? wslDistro) && currentPath.Equals(wslDistro.Path))
+			{
+				tabLocationHeader = wslDistro.Text;
+				iconSource.ImageSource = new Microsoft.UI.Xaml.Media.Imaging.BitmapImage(wslDistro.Logo);
```

Comment:
Can the namespace be simplified?

Reply:
I included the namespace, should be fixed now